connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -691,6 +691,46 @@ class SparkConnectPlanner(session: SparkSession) {
         }
         Some(NthValue(children(0), children(1), ignoreNulls))
 
+      case "window" if 2 <= fun.getArgumentsCount && fun.getArgumentsCount <= 4 =>
```

Comment:
TimeWindow and SessionWindow do not have suitable constructor for the string parameters